tests/components/lcn/test_config_flow.py

```diff
@@ -21,14 +23,19 @@
 
 from tests.common import MockConfigEntry
 
-IMPORT_DATA = {
-    CONF_HOST: "pchk",
+OPTIONS_DATA = {
     CONF_IP_ADDRESS: "127.0.0.1",
-    CONF_PORT: 4114,
+    CONF_PORT: 1234,
```

Comment:
Why is this changed?

Reply:
I need to check the reconfiguration_step in the corresponding test. That's why I simply changed the port.